mo-helloworld/hello.go

```diff
@@ -0,0 +1,8 @@
+# Simple program for teaching git flow and code review
+package main
+
+import "fmt"
+
+func main() {
```

Comment:
Why on earth did you name this function main?

Reply:
because if I didn't I couldn't run the code :shipit: ?

Reply:
we should implement a python version tho

Reply:
You reasoning is sound, but ultimately I agree with you that a Python version would be better.